app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>List of all users</h1>
\ No newline at end of file
+<ul class="userCards py-8 w-[80%] mx-auto flex flex-col gap-8"><div class='row>
+<% @users.each do |user| %>
+     <li class="userCard">
+        <a href="users/<%= user.id %>" class="flex flex-row gap-[30px]">
+            <div class="userImage w-[15%]">
+                <img src='<%= user.photo %>'>
+            </div>
+            
+        <div class="userDetails bg-[#2B3A55] rounded-[30px] flex-grow text-white flex justify-between items-center p-[30px]">
+          <h2 class="text-[40px] font-[600]"><%= user.name %></h2>
+          <p class="self-end text-[20px] font-[400]">Number of posts: <%= user.posts_counter %></p>
+        </div>
+        </a>
```

Comment:
- [ ] The user name should be clickable and when I click on a user name it should redirect to the user's show page. Kindly do that.